app/src/lib/databases/repositories-database.ts

```diff
@@ -33,6 +34,7 @@ export interface IDatabaseGitHubRepository {
   readonly isArchived?: boolean
 
   readonly permissions?: 'read' | 'write' | 'admin' | null
+  readonly organization?: IAPIOrganization
```

Comment:
We shouldn't be storing raw API objects in the database. They often contain a lot more dat than we need and we can't trust that their shape stays consistent across GHE/GHES/GHAE versions. In the case of Organization it's not too bad:

```json
"organization": {
		"login": "desktop",
		"id": 13171334,
		"node_id": "MDEyOk9yZ2FuaXphdGlvbjEzMTcxMzM0",
		"avatar_url": "https://avatars.githubusercontent.com/u/13171334?v=4",
		"gravatar_id": "",
		"url": "https://api.github.com/users/desktop",
		"html_url": "https://github.com/desktop",
		"followers_url": "https://api.github.com/users/desktop/followers",
		"following_url": "https://api.github.com/users/desktop/following{/other_user}",
		"gists_url": "https://api.github.com/users/desktop/gists{/gist_id}",
		"starred_url": "https://api.github.com/users/desktop/starred{/owner}{/repo}",
		"subscriptions_url": "https://api.github.com/users/desktop/subscriptions",
		"organizations_url": "https://api.github.com/users/desktop/orgs",
		"repos_url": "https://api.github.com/users/desktop/repos",
		"events_url": "https://api.github.com/users/desktop/events{/privacy}",
		"received_events_url": "https://api.github.com/users/desktop/received_events",
		"type": "Organization",
		"site_admin": false
	},
```

Regardless I think a better approach here is to add an optional string "type" property to `IDatabaseOwner`. If you look at the API response for say desktop/desktop you can determine that the owner is an organization by looking at `owner.type`. See

```json
{
	...
	"name": "desktop",
	"full_name": "desktop/desktop",
	...
	"owner": {
		"login": "desktop",
		...
		"type": "Organization",
		...
	},
```

It'll be a little bit more work but it'll be worth it. I recently did some work on saving owners with correct casing so I've got this pretty fresh in my memory if you want to pair on adding it.

Reply:
> shouldn't be storing raw API objects in the database. They often contain a lot more dat tha

:flushed: That totally makes sense... not that you point it out. 

I didn't know about the owner having the type on it (follow dotcom logic, the repo was checked for org). This follows more logically with what I am trying to accomplish. 🙇 

I have updated to change to adding an optional owner type so it can be added as the repo is refreshed. Even if there is a case where a pr is viewed before this happens, missing a link isn't isn't a big let down and should only happen once.